nixos/hardware/COPYING

```diff
@@ -0,0 +1,111 @@
+Statement of Purpose
```

Comment:
This is a public domain license. So, for simplicity's sake, I'd remove it and keep a top-level MIT license.

Reply:
I wasn’t sure of that, as IANAL I opted for keeping it safe. I would make that decision in a separate PR.